misago/users/models/deleteduser.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+
+DELETED_BY_USER = 1
```

Comment:
All those constants should be class attributes of `DeletedUser`, so they are easily reachable like this: `DeletedUser.BY_STAFF`